backend/api/Areas/Property/Mapping/Building/BuildingMap.cs

```diff
@@ -65,14 +65,14 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.Parcels, src => src.Parcels)
                 .AfterMapping((src, dest) =>
                 {
-                    if(src.LeasedLandMetadata == null)
+                    if (src.LeasedLandMetadata == null)
                     {
                         dest.LeasedLandMetadata = new List<Model.LeasedLandMetadataModel>();
                         return;
                     }
-                    var metadata = JsonSerializer.Deserialize<IEnumerable<Entity.Models.LeasedLandMetadata>>(src.LeasedLandMetadata, _serializerOptions);
+                    var metadata = JsonSerializer.Deserialize<IEnumerable<Entity.Models.LeasedLandMetadataModel>>(src.LeasedLandMetadata, _serializerOptions);
 
-                    dest.LeasedLandMetadata = metadata.Where(m => m != null).Select(l => new Model.LeasedLandMetadataModel { OwnershipNote = l.OwnershipNote, ParcelId = l.ParcelId, Type = l.Type });
+                    dest.LeasedLandMetadata = metadata.Where(m => m != null).Select(l => new Model.LeasedLandMetadataModel { OwnershipNote = l.OwnershipNote, ParcelId = l.ParcelId, Type = (int)l.Type });
```

Comment:
Needs to be converted to an `int` for the frontend code.  Most of our `enum` values are converted to their string names, but here they are not.